app/src/main/java/com/example/taskwise_eventmaster/presentation/home_page/ProfilePartOfHomeScreen.kt

```diff
@@ -38,10 +38,10 @@ import kotlinx.coroutines.launch
 
 @Composable
 fun ProfilePart(
```

Comment:
Let's rename this to just `Profile` as `part` doesn't bring much value to the meaning